plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php

```diff
@@ -71,6 +71,12 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		KalturaLog::debug("checking entry id - $entryId");
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);

```

Comment:
@hilak I thought it's more elegant getting it through the plugin, but basically I agree. I'll handle it if i have another round of code fixes here.